lib/Wikia/src/Vignette/UrlGenerator.php

```diff
@@ -31,6 +31,10 @@ class UrlGenerator {
 
 	const REVISION_LATEST = 'latest';
 
+	const BASE_URL_PROD = 'http://vignette<SHARD>.wikia.nocookie.net';
```

Comment:
Hmm. Okay, well, maybe it makes the most sense to just put 'http://vignette<SHARD>.wikia.nocookie.net' in the test itself. Looking through the rest of the config repo, I don't see it referencing the app repo anywhere else. It does end up repeating some code, but it keeps all of our config in the config repo and doesn't tie to /lib/Wikia to MW.